src/main/java/com/hundred/percent/capstone/Invoicify/invoice/controller/InvoiceController.java

```diff
@@ -0,0 +1,29 @@
+package com.hundred.percent.capstone.Invoicify.invoice.controller;
+
+
+import com.hundred.percent.capstone.Invoicify.invoice.dto.InvoiceDTO;
+import com.hundred.percent.capstone.Invoicify.invoice.service.InvoiceService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/invoices")
+public class InvoiceController {
+
+    @Autowired
+    InvoiceService invoiceService;
+
+    @GetMapping
+    public List<InvoiceDTO> getAllInvoices(){
+        return invoiceService.getAllInvoices();
+    }
+
+    @PostMapping
```

Comment:
consider adding an endpoint that explicitly say that you are adding an invoice 
ex: /addInvoice